REPL.h

```diff
@@ -22,13 +23,13 @@
 
 struct REPL
 {
-    static llvm::Expected<std::unique_ptr<REPL>> Create();
+    static llvm::Expected<std::unique_ptr<REPL>> Create(bool is_playground = false);
     std::string GetLine();
     bool IsExitString(const std::string &line);
     bool ExecuteSwift(std::string line);
 
 protected:
-    REPL();
+    REPL(bool is_playground);
```

Comment:
You should probably mark the constructor as `explicit`.  You don't want a `bool` to `REPL` implicit conversion.